third_party/nvfuser/csrc/fusion_segmenter.cpp

```diff
@@ -1949,7 +1952,7 @@ bool TranslateApplicableWelford::wouldTranslateToPersistent(
   return isValidPersistentFusion(test_copy.get(), runtime_info);
 }
 
-void TranslateApplicableWelford::translateSingleWelford(WelfordOp* welford) {
+void TranslateApplicableWelford::translateSingleWelford(WelfordOp* welford, bool reuse_x_mean_sub) {
```

Comment:
Do you think there's a case where this flag should be `true`? If not let's delete it.